### PR TITLE
[ACS-8036] [Bulk update] [Many files to 1 Hold flow] "Manage Holds" dialog with Manage Existing Holds/Apply New Hold tabs

### PR DESCRIPTION
**Pull request by DaryaBalvanovich**
**Please check if the PR fulfills these requirements**

https://hyland.atlassian.net/browse/ACS-8036

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by MichalKinas — APPROVED** [review]

File: projects/aca-content/src/lib/components/dl-custom-components/datatable-cell-badges/datatable-cell-badges.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      providers: [Actions]
+    });
+
+    fixture = TestBed.createComponent(DatatableCellBadgesComponent);
+    component = fixture.componentInstance;
+    appExtensionService = TestBed.inject(AppExtensionService);
+    component.node = mockNode;
+  });
+
+  it('should get badges when component initializes', async () => {
```
Comment (swapnil-verma-gl) [inline projects/aca-content/src/lib/components/dl-custom-components/datatable-cell-badges/datatable-cell-badges.component.spec.ts:88]: async is not needed here

Comment (g-jaskowski) [reply projects/aca-content/src/lib/components/dl-custom-components/datatable-cell-badges/datatable-cell-badges.component.spec.ts]: removed, thanks!

**Comment by g-jaskowski** [comment]
SonarCloud accessibility errors are a false positive

File: projects/aca-content/src/lib/components/dl-custom-components/datatable-cell-badges/datatable-cell-badges.component.spec.ts (hunk 1 above)
Comment (DaryaBalvanovich) [reply projects/aca-content/src/lib/components/dl-custom-components/datatable-cell-badges/datatable-cell-badges.component.spec.ts]: fixed, thanks for catching!

**Review by nikita-web-ua — APPROVED** [review]

**Review by dominikiwanekhyland — APPROVED** [review]

**Review by tamaragruszka — APPROVED** [review]

**Pull request merged**